### PR TITLE
fix: ts-ignore langchain core and langchain errors

### PR DESCRIPTION
**Pull request by paul-paliychuk**
> :rocket: This PR description was created by [Ellipsis](https://www.ellipsis.dev) for commit e1d554ba5a146e0eb899998fee5d44306ae4a9c9.

### Summary:
This PR suppresses TypeScript compiler warnings by adding `ts-ignore` comments in two files: `message_history_vector_store_example.ts` and `vector_store_example.ts`.

**Key points**:
- Added `ts-ignore` comments in `message_history_vector_store_example.ts` and `vector_store_example.ts`.
- The comments are added before the `formatDocument` function call in the `combineDocuments` function and before the `model` pipe in the `main` function.

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

File: examples/langchain/vector_store_example.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ async function combineDocuments(
 ) {
    const docStrings: string[] = await Promise.all(
       docs.map((doc) => {
+         // @ts-ignore
          return formatDocument(doc, documentPrompt);
```
[the remark this answers is not in the document]
Comment (paul-paliychuk) [reply examples/langchain/vector_store_example.ts:29]: The code is functional, it's an issue with langchain and langchain core compatibility

File: examples/langchain/message_history_vector_store_example.ts [hunk 2]
```diff
@@ -79,6 +80,7 @@ async function main() {
 
    const ragChain = setupAndRetrieval
       .pipe(prompt)
+      // @ts-ignore
       .pipe(model)
```
[the remark this answers is not in the document]
Comment (paul-paliychuk) [reply examples/langchain/message_history_vector_store_example.ts:84]: The code is functional, it's an issue with langchain and langchain core compatibility

[the remark this answers is not in the document]
Comment (paul-paliychuk) [reply examples/langchain/message_history_vector_store_example.ts:84]: The code is functional, it's an issue with langchain and langchain core compatibility

**Comment by danielchalef** [comment]
@paul-paliychuk why is this PR needed? Are these peer deps or required?

**Pull request closed without merging**